lv_conf_template.h

```diff
@@ -26,22 +26,38 @@
    COLOR SETTINGS
  *====================*/
 
-/*Color depth: 1 (I1), 8 (L8), 16 (RGB565), 24 (RGB888), 32 (XRGB8888)*/
+/** Color depth: 8 (A8), 16 (RGB565), 24 (RGB888), 32 (XRGB8888) */
```

Comment:
Keep I1 and L8
```suggestion
/** Color depth: 1 (I1), 8 (L8), 16 (RGB565), 24 (RGB888), 32 (XRGB8888) */
```

Reply:
Good catch, @kisvegabor !  I hadn't realized this has changed for v9.2!  This inspired me to also look to find if there were any more recent changes to `lv_conf_template.h` for v9.2 that I might have overlooked, and I found one more -- the comments to the right of the LV_WAYLAND_W... macros.  Also fixed.